packaging/linux/Arch/sunshine.install

```diff
@@ -2,11 +2,19 @@ do_setcap() {
   setcap cap_sys_admin+p $(readlink -f $(which sunshine))
 }
 
+do_udev_reload() {
+  udevadm control --reload-rules
+  udevadm trigger --property-match=DEVNAME=/dev/uinput
+  modprobe uinput || true
```

Comment:
```suggestion
  modprobe uinput || echo "Warning: Failed to reload udev rules, a system restart is required!"
```

I don't know if anything is output to the user, but if so maybe something like this would be better?

Reply:
It's possible for `uinput` to not be built as a module in which case the `modprobe uinput` is not necessary.

Reply:
Okay, I'm fine with what you had as well.